spec/controllers/home_controller_spec.rb

```diff
@@ -1,2 +1,82 @@
 require 'rails_helper'
 
+RSpec.describe HomeController, type: :controller do
+  describe 'GET #index' do
+    before { get :index }
+
+    it 'should redner the right partial' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'should get a hash of the top 10 reddit posts' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.